tests/test_write_block.py

```diff
@@ -46,7 +50,14 @@ def make_blocks(num_records=2000, codec='null'):
     return blocks, records
 
 
-def check_concatenate(source_codec='null', output_codec='null'):
+@pytest.mark.parametrize(
```

Comment:
Thanks! This is definitely better than the way the test was previously being done.